suite-common/test-utils/src/conditionalDescribe.ts

```diff
@@ -4,8 +4,10 @@ export const conditionalDescribe = (
     fn: jest.EmptyFunction,
 ) => {
     if (skipCondition) {
+        // eslint-disable-next-line jest/valid-describe-callback
```

Comment:
Seems to be the same case as `jest/valid-title`, I'd disable this rule globally.